testsuite/integration/clustering/src/test/java/org/jboss/as/test/clustering/arquillian/KillAllContainersOnAfterSuiteExtension.java

```diff
@@ -34,27 +33,26 @@
  *
  * @author <a href="mailto:aslak@redhat.com">Aslak Knutsen</a>
  * @author Radoslav Husar
- * @version $Revision: $
+ * @version October 2015
  */
-public class StopCustomContainersOnAfterSuiteExtension implements LoadableExtension {
+public class KillAllContainersOnAfterSuiteExtension implements LoadableExtension {
 
-    private static final Logger log = Logger.getLogger(StopCustomContainersOnAfterSuiteExtension.class);
+    private static final Logger log = Logger.getLogger(KillAllContainersOnAfterSuiteExtension.class);
 
     @Override
     public void register(ExtensionBuilder builder) {
-        builder.observer(StopCustomContainers.class);
+        builder.observer(KillAllContainers.class);
     }
 
-    public static class StopCustomContainers {
+    public static class KillAllContainers {
+        @SuppressWarnings("UnusedParameters")
         public void close(@Observes AfterSuite event, ContainerRegistry registry) {
-            for (Container c: registry.getContainers()) {
-                if (c.getState() == State.STARTED && "custom".equalsIgnoreCase(c.getContainerConfiguration().getMode())) {
-                    try {
-                        c.stop();
-                        log.info("Stopped custom container: " + c.getName());
-                    } catch (Exception e) {
-                        log.error("Failed to stop custom container: " + c.getName(), e);
-                    }
+            for (Container container : registry.getContainers()) {
```

Comment:
As I am looking though wildfly-arquillian, it relies on a fact to implement org.jboss.arquillian.container.spi.ServerKillProcessor for kill() to actually work properly. If not implemented, it pretty much calls stop() per org.jboss.arquillian.container.impl.DefaultServerKillProcessor.